src/components/Chip.tsx

```diff
@@ -20,13 +20,14 @@ export interface ChipProps<X> {
   title?: string;
   xss?: X;
   type?: ChipType;
+  compact?: boolean;
 }
 
 /** Kinda like a chip, but read-only, so no `onClick` or `hover`. */
 export function Chip<X extends Only<Xss<Margin>, X>>({ type = ChipTypes.neutral, ...props }: ChipProps<X>) {
-  const { text, title = text, xss = {} } = props;
+  const { text, title = text, xss = {}, compact: propsCompact } = props;
```

Comment:
Maybe we can move this code around and avoid the `compact: propsCompact` and have it default to `fieldProps.compact`. Something like:
```
const { fieldProps } = usePresentationContext();
const { text, title = text, xss = {}, compact = fieldProps?.compact } = props;
```